asdf_astropy/converters/unit/equivalency.py

```diff
@@ -0,0 +1,36 @@
+from asdf.extension import Converter
+
+
+class EquivalencyConverter(Converter):
+    tags = ["tag:astropy.org:astropy/units/equivalency-*"]
+
+    types = ["astropy.units.equivalencies.Equivalency"]
+
+    def to_yaml_tree(self, obj, tag, ctx):
+        results = []
+
+        for name, kwargs in zip(obj.name, obj.kwargs):
+            kwargs_names = []
+            kwargs_values = []
+            for kwargs_name, kwargs_value in kwargs.items():
+                kwargs_names.append(kwargs_name)
+                kwargs_values.append(kwargs_value)
+
+            results.append({
+                "name": name,
+                "kwargs_names": kwargs_names,
+                "kwargs_values": kwargs_values,
+            })
+
+        return results
```

Comment:
```suggestion
        return [
            {'name': name, 'kwargs_names': list(kw.keys()), 'kwargs_values': list(kw.values())}
            for name, kw in zip(obj.name, obj.kwargs)
        ]
```

Reply:
Are `keys()` and `values()` guaranteed to return items in the same order when called independently?  I assumed not, but I don't know one way or the other...

Reply:
Oh, right, yes, dicts are ordered now.  Thanks, this is much tidier.

Reply:
@perrygreenfield What do you think?

Reply:
From https://docs.python.org/3/library/stdtypes.html#dict-views:

> The objects returned by dict.keys(), dict.values() and dict.items() are view objects. They provide a dynamic view on the dictionary’s entries, which means that when the dictionary changes, the view reflects these changes.

and

> Keys and values are iterated over in insertion order. This allows the creation of (value, key) pairs using zip(): pairs = zip(d.values(), d.keys()).